core/jazz_acl/config/rbac_model.conf

```diff
@@ -0,0 +1,14 @@
+[request_definition]
+r = sub, obj, act
+
+[policy_definition]
+p = sub, obj, act
+
+[role_definition]
+g = _, _
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m=g(r.sub, p.sub) && keyMatch(r.obj, p.obj) && (r.act == p.act || (p.act == "write" || p.act == "admin"))
```

Comment:
What does this justify - `(r.act == p.act || (p.act == "write" || p.act == "admin"))`? A note/comment might help (if adding a comment is feasible here)

Reply:
if the user and service_category matches with the request and the user has write/admin permission then they automatically get read permission too. Also added as comment to the code